### PR TITLE
[fa]: Update translations

### PR DESCRIPTION
**Pull request by AzadGh95**
change فایل to مورد
it's more correct

### CODE REVIEW

**Review by andrey-helldar — APPROVED** [review]
It's ok for me

**Comment by ariaieboy** [comment]
> It's ok for me

line 59 must be reverted but other changes are good.

File: locales/fa/json.json [hunk 1]
```diff
@@ -55,16 +55,16 @@
     "Apply": "تایید",
     "Apply Coupon": "تایید کوپن",
     "April": "آپریل",
-    "Are you sure you want to delete the selected resources?": "آیا مطمئن هستید که می‌خواهید فایل های انتخاب شده را حذف کنید؟",
-    "Are you sure you want to delete this file?": "آیا از حذف این فایل مطمئن هستید؟",
+    "Are you sure you want to delete the selected resources?": "آیا مطمئن هستید که می‌خواهید موارد انتخاب شده را حذف کنید؟",
+    "Are you sure you want to delete this file?": "آیا از حذف این مورد مطمئن هستید؟",
```
Comment (andrey-helldar) [inline locales/fa/json.json:59]:
```suggestion
    "Are you sure you want to delete this file?": "آیا از حذف این فایل مطمئن هستید؟",
```

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Review by andrey-helldar — APPROVED** [review]
ممنون

**Pull request merged**